tema1.css

```diff
@@ -0,0 +1,37 @@
+
+.form {
+    color:skyblue;
+}
+
+.p {
+  border-color: green;
+}
```

Comment:
In html trebuia sa folosesti `<label>` astfel incat sa selectezi toate `label` in CSS.
Cum ai procedat aici merge, dar nu e corect conform cerintei